### PR TITLE
miscellaneous changes

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: larray/viewer/view.py [hunk 1]
```diff
@@ -1316,8 +1316,9 @@ def setup_menu_bar(self):
                                           triggered=self.close))
 
         help_menu = menu_bar.addMenu('&Help')
-        help_menu.addAction(create_action(self, _('Online documentation'), shortcut="Ctrl+H",
-                                          triggered=self.open_documentation))
+        help_menu.addAction(create_action(self, _('Online &Documentation'), shortcut="Ctrl+H", triggered=self.open_documentation))
+        help_menu.addAction(create_action(self, _('&Getting Started'), triggered=self.open_tutorial))
```
Comment (gdementen) [inline larray/viewer/view.py]: why not name this "Tutorial" or "Online Tutorial" instead of "Getting Started"?

File: larray/viewer/view.py [hunk 2]
```diff
@@ -1316,8 +1316,9 @@ def setup_menu_bar(self):
...
+        help_menu.addAction(create_action(self, _('Objects and &Functions'), triggered=self.open_api_documentation))
```
Comment (gdementen) [inline larray/viewer/view.py]: Objects and Functions (API) &Reference?

Comment (gdementen) [inline larray/viewer/view.py]: Online?

File: larray/viewer/view.py (hunk 1 above)
Comment (alixdamman) [inline larray/viewer/view.py]: IMO, Getting Started is the equivalent of 10 minutes to Pandas which does not correspond to the "tutorial". As I see it, the getting started should be read in maximum 10 minutes and just present the main features of LArray

Comment (alixdamman) [inline larray/viewer/view.py]: Well, here open_tutorial will open the tutorial. So you're right. Sorry

File: larray/viewer/view.py (hunk 2 above)
Comment (alixdamman) [inline larray/viewer/view.py]: Our main users don't know what API means. That's why I didn't mentioned it. But OK, I could add in ( ).

File: larray/viewer/view.py (hunk 1 above)
Comment (alixdamman) [inline larray/viewer/view.py]: I mixed up with PR #329

**Comment by alixdamman** [comment]
updated

**Comment by gdementen** [comment]
LGTM

**Comment by gdementen** [comment]
I know... But for those who know what that means, it is clearer. Or at
least I usually scan for both "API" and "Reference" keywords. Here there
was neither.

**Pull request merged**